zerver/tests/test_email_change.py

```diff
@@ -175,7 +175,7 @@ def test_end_to_end_flow(self) -> None:
         self.assertEqual(self.email_envelope_from(email_message), settings.NOREPLY_EMAIL_ADDRESS)
         self.assertRegex(
             self.email_display_from(email_message),
-            rf"^Zulip Account Security <{self.TOKENIZED_NOREPLY_REGEX}>\Z",
+            rf"^{settings.SERVICE_NAME} account security <{self.TOKENIZED_NOREPLY_REGEX}>\Z",
```

Comment:
It's better for tests to verify concrete values; the only reason we need matching here is because `TOKENIZED_NOREPLY_REGEX` matches with a nondeterministic value.